sources/errors.ts

```diff
@@ -6,6 +6,27 @@ export type ErrorMeta = {
   type: `usage`;
 };
 
+/**
+ * An error with metadata telling clipanion how to print it
+ *
+ * Errors with this metadata property will have their name and message printed, but not the
+ * stacktrace.
+ *
+ * This should be used for errors where the message is the part that's important but the stacktrace is useless.
+ * Some examples of where this might be useful are:
+ *
+ * - Invalid input by the user (see `UsageError`)
+ * - A HTTP connection fails, the user is shown "Failed To Fetch Data: Could not ocnnect to server example.com" without stacktrace
```

Comment:
```suggestion
 * - A HTTP connection fails, the user is shown "Failed To Fetch Data: Could not connect to server example.com" without stacktrace
```

Reply:
I'd pay good money for a decent spell check plugin for VS Code, because the number of typos I commit is too damn high.

Thanks for catching this one!